addons/escoria-core/game/core-scripts/esc/esc_object_manager.gd

```diff
@@ -483,14 +455,12 @@ func _room_exists(room_key: ESCRoomObjectsKey) -> bool:
 #
 # - object: The object to check for existence.
 # - room_key: The key representing the desired room in the object manager array.
-# **Returns** True iff object exists in the object manager entry specified by room_key.
```

Comment:
Please put this back. "iff" isn't a typo; it's a common shorthand for "if and only if", which I've used on purpose.

Reply:
oops! Alright then, sorry about that I had no idea of this :D